cppcache/src/Log.cpp

```diff
@@ -382,7 +383,8 @@ std::string Log::formatLogLine(LogLevel level) {
       << std::put_time(&tm_val, "%Y/%m/%d %H:%M:%S") << '.' << std::setfill('0')
       << std::setw(6) << microseconds.count() << ' '
       << std::put_time(&tm_val, "%z  ") << g_hostName << ":"
-      << boost::this_process::get_id() << " " << std::this_thread::get_id()
+      << boost::this_process::get_id() << " "
+      << DistributedSystemImpl::getThreadName(std::this_thread::get_id())
```

Comment:
I am not totally sure about this, but changing the logging format might break the GNMSG tool that @pdxcodemonkey implemented some time ago. I am not sure what level of support you are intending to provide for this tool, but might be  good to make sure that the tool could handle both log formats

Reply:
Good catch. gnmsg has been updated to handle either kind of log lines as below:
```[debug 2022/02/06 07:44:02.450497 -0800  FirstPro:20336 18736] EventId::EventId(000002894033BF60) - doInit=false, reserveSize=0, fullValueAfterDeltaFail=false
[fine 2022/02/06 07:44:02.453857 -0800  FirstPro:20336 18812 (NC ETM Thread)] ExpiryTaskManager thread is running.```